06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/index.js

```diff
@@ -0,0 +1,53 @@
+class InvertedIndex {
+
+  constructor() {
+    this.textArray = [];
+    this.index = {};
+  };
+
+  transformToSingles(file) {
+    for (let words in file) {
+      file[words] = file[words].replace(/'\w+\s/g, " ").replace(/[.,/#!+$%^&@*?;:'{}=\-_`~()]/g, '').trim().toLowerCase().split(' ')

```

Comment:
No it wasn't. I was having serious issues concentrating when there were red lines everywhere. I'll definitely do better in the next CP.